### ISSUE TITLE
Imported text fields with values consisting of digits gets treated as an actual number

### ISSUE THREAD

**Issue by seav** [Cadasta/cadasta-platform#1758]
### Steps to reproduce the error
1. Create a new project with the following questionnaire: 
[numeric_text_questionnaire.xlsx](https://github.com/Cadasta/cadasta-platform/files/1263919/numeric_text_questionnaire.xlsx)
2. Import the following data file (select just the "Party" import type): 
[numeric_test_import.xlsx](https://github.com/Cadasta/cadasta-platform/files/1263939/numeric_test_import.xlsx)
3. Go to the Parties tab and view the Individual party and the Corporation party

### Actual behavior
The National Id Number, Mobile Number, and Tax ID fields, which are text fields, have decimal places (".0").

![screen shot 2017-08-30 at 22 34 11](https://user-images.githubusercontent.com/873653/29877856-6016a620-8dd3-11e7-8cd9-a072249ab955.png)

![screen shot 2017-08-30 at 22 33 46](https://user-images.githubusercontent.com/873653/29877859-631ae6b0-8dd3-11e7-8bee-6e732308389a.png)

### Expected behavior
There shouldn't any be decimal places.

### Notes
- This is pretty weird, actually. If you look at the import XLSX file, the cells containing those values are treated as text. The import file is actually just an export file based from data entered via the UI.

**Comment by dpalomino** [comment]
Wow, it is weird yes. Because in the questionnaire these fields are also `text`s. Any idea why is this happening? 

@SteadyCadence, have you seen something like that before  when importing data?

**Comment by dpalomino** [comment]
I am assigning a high priority because I think this could complicate the import of data....

**Comment by SteadyCadence** [comment]
This issue sounds a lot like https://github.com/Cadasta/cadasta-platform/issues/1231

**Comment by SteadyCadence** [comment]
I havent had any issues with it, but I am will be using the API for big imports...

**Comment by SteadyCadence** [comment]
I think its more of an excel issue because when I convert the import file to csv those numbers are converted to integers. They are not wrapped in strings.

**Comment by oliverroick** [comment]
This issue is caused by a library we're using to read spreadsheets. It automatically converts any string that is a number to a float. This only affects imports, not the web site, not the API. If you provide something that isn't a number it is treated as text. So it doesn't complicate import of data, the information just isn't represented correctly. 

Question: Is this really a high priority? This has been around for a while now and people didn't seem too bothered by it. When you define a field as text, you would expect an alpha-numeric input. The social security numbers and tax IDs I had in my life (in three countries) never were just numbers. It's likely that we never run into these problems in a real world scenario.

**Comment by SteadyCadence** [comment]
@dpalomino @oliverroick I don't think it is a high priority. With the partners I am working with, we can work around those issues.

**Comment by dpalomino** [comment]
Hi @oliverroick @SteadyCadence 

Well, I think the main problem would be with phone numbers. We can certainly workaround that but, it complicates the import process (and likely prevent a regular partner to import phone numbers by their own). Not a blocking issue of course, but I think it'd be useful to solve. Re-prio to medium.
